packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -31,7 +31,7 @@ import {
 import { getSansRegular } from '@bbc/psammead-styles/font-styles';
 
 // Transparent border is to show edge of component and link underline on hover/focus in high-contrast mode
```

Comment:
Can we updaet this comment please?

Reply:
Thanks for the suggestion @greenc05, would "Transparent borders are used to highlight components and their borders in windows high-contrast mode." work?

Reply:
That's probably more appropriate for the a11y section of the read me? The const is just used for the top of the component and the button isn't it? Not the links? That's what I was referring to updating...

Reply:
Ah, thanks @greenc05, makes sense. I'll update now.